rfcs/theming-color-scheme-strategy.md

```diff
@@ -0,0 +1,164 @@
+Theming Strategy in Orbit
+
+## Summary
+
+Orbit has been, in it's infancy, a monolitich piece of CSS, the only theming options were to modify the primary colors in order to adapt to each products in the Sharegate ecosystem(Apricot, Overcast and Desktop). With Orbit 2.0 we aim to support, among things, a dark theme, while opening the door for future theming possibilitiess(high Contrast, density themes). In this RFC we propose a new architecture for CSS Custom Properties and a way to organize them.
+
+## Motivation
+
+Being able to quickly add a design token, and semantic variables, without questionning where to do so, is a main goal of ours. This is why we came up with a strategy, and are moving many custom properties. In order to provide a consistent theme, that behaves in a predictable manne, we are introducing the concept of semantic colors. Semantic colors are a layer over design tokens representing a meaning, e.g. `border-danger` has a semantic meaning while `border-red-500` has none and could be used on many situations. This make our intefaces more cohesive and gives intention to our design choices. As Adobe Spectrum sums it up:
```

Comment:
in a predictable manner

Reply:
Where would you add this: > in a predictable manner

Reply:
There is a typo, `that behaves in a predictable manne` should be `manner`